.rubocop_local.yml

```diff
@@ -4,4 +4,4 @@ Style/GlobalVars:
 
 Style/MethodCallWithArgsParentheses:
   Exclude:
-  - spec/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/vm_spec.rb
+  - spec/models/manageiq/providers/ibm_cloud/vpc/**/*.rb
```

Comment:
Is this required?  I deleted this cop override entirely and don't see any offenses:
```
$ git diff
diff --git a/.rubocop_local.yml b/.rubocop_local.yml
index 268fbb2..77bf231 100644
--- a/.rubocop_local.yml
+++ b/.rubocop_local.yml
@@ -1,7 +1,3 @@
 Style/GlobalVars:
   AllowedVariables:
   - $ibm_cloud_log # Global cloud plugin log from ManageIQ.
-
-Style/MethodCallWithArgsParentheses:
-  Exclude:
-  - spec/models/manageiq/providers/ibm_cloud/vpc/**/*.rb
```

```
$ bundle exec rubocop spec/models/manageiq/providers/ibm_cloud/vpc/

Inspecting 4 files
....

4 files inspected, no offenses detected
```

Reply:
I see it in my linter, thus the reason I modified it. Leave it, please. Your rubocop may not be the same version as mine. I don't want this cop to show up in my linter.

Reply:
Your linter should be pulling in the base rubocop yml config though so my linter config is the same as miq-bot's should be the same as your.

Reply:
Not sure what linter you are using @jared-ibm, but ideally you should use the one that is used by @miq-bot so that we are all seeing the same thing.  This is probably why you are not seeing https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/137#issuecomment-780853035 in your personal linter.

Reply:
I am using 1.3.1 which is from Nov 16, 2020. The current is 1.10, so both of us are behind.

Reply:
The bot is using 0.82.0, which is what we should all use for consistency.  To get the right version you just need to run `bundle exec rubocop` since the manageiq-style gem drives the rubocop version both for the projects and for the bot, so they are in sync.  There are issues with inconsistencies in the configuration between rubocop versions which is why it's important we all match.

Reply:
Ummm, you are telling us that we should be able to react to the changes in the automated SDKs but we should also stay at an older version of a secondary check doesn't really work for me. I have no problem keeping up with SDK changes, just as I have no problem with keeping up with newer versions of rubocop, or ruby for that matter.